Storage/Storage/Model/Copiable/Models+Copiable.generated.swift

```diff
@@ -60,18 +60,21 @@ extension GeneralStoreSettings {
         isTelemetryAvailable: CopiableProp<Bool> = .copy,
         telemetryLastReportedTime: NullableCopiableProp<Date> = .copy,
         areSimplePaymentTaxesEnabled: CopiableProp<Bool> = .copy,
-        preferredInPersonPaymentGateway: NullableCopiableProp<String> = .copy
+        preferredInPersonPaymentGateway: NullableCopiableProp<String> = .copy,
+        skippedCodOnboardingStep: CopiableProp<Bool> = .copy
```

Comment:
Not a blocker either, I think it would be more clear if we use the full `CashOnDelivery` wording instead of the acronym Cod 🐟 😅